cmake/AlembicPyIlmBase.cmake

```diff
@@ -36,9 +36,17 @@
 FIND_PACKAGE(PyIlmBase)
 
 IF (PYILMBASE_FOUND)
-    SET(ALEMBIC_PYILMBASE_INCLUDE_DIRECTORY ${ALEMBIC_PYILMBASE_INCLUDE_DIRECTORY})
-    SET(ALEMBIC_PYILMBASE_LIBRARIES ${ALEMBIC_PYILMBASE_LIBS})
     SET(ALEMBIC_PYILMBASE_FOUND 1 CACHE STRING "Set to 1 if PyIlmBase is found, 0 otherwise")
+
+    SET(ALEMBIC_PYILMBASE_LIBS
+        ${ALEMBIC_PYILMBASE_PYIMATH_LIB}
+    )
+
+    GET_FILENAME_COMPONENT(ALEMBIC_PYILMBASE_LIB_DIRECTORY
+        ${ALEMBIC_PYILMBASE_PYIMATH_LIB}
+        DIRECTORY
```

Comment:
Per:
https://cmake.org/cmake/help/v3.0/command/get_filename_component.html

Can you use PATH here instead of DIRECTORY for backwards compatibility with older CMakes?
How badly does your verson of CMake complain?